ansible/config_sonic_basedon_testbed.yml

```diff
@@ -74,6 +74,101 @@
         vm_base: "{% if testbed_facts['vm_base'] != '' %}{{ testbed_facts['vm_base'] }}{% else %}''{% endif %}"
     when: testbed_name is defined
 
+  - name: Run L1 Switch connection panel for RDMA testing
+    block:
```

Comment:
Looks like this block of code is also in file `restore_l1switch_connections.yml`. Use `include_tasks` or similar method to avoid code duplication?